packages/cache/__tests__/tar.test.ts

```diff
@@ -149,17 +149,17 @@ test('zstd create tar', async () => {
     `"${tarPath}"`,
     [
       '--posix',
-      '--use-compress-program',
-      IS_WINDOWS ? 'zstd -T0 --long=30' : 'zstdmt --long=30',
-      '-cf',
-      IS_WINDOWS ? CacheFilename.Zstd.replace(/\\/g, '/') : CacheFilename.Zstd,
       '--exclude',
       IS_WINDOWS ? CacheFilename.Zstd.replace(/\\/g, '/') : CacheFilename.Zstd,
       '-P',
       '-C',
       IS_WINDOWS ? workspace?.replace(/\\/g, '/') : workspace,
       '--files-from',
-      'manifest.txt'
+      'manifest.txt',
```

Comment:
Why the reorder?

Reply:
Tests were failing so the order needs be the same here as well.